File: problem-1/problem_1.c
#include <stdio.h>

int main(int argc, char *argv[]){
  
  int i = 0;
  int sum = 0;

  while(i < 1000){
    if(i % 3 == 0 || i % 5 == 0){
      sum += i;
    }
    i++;
  }

  printf("Sum of multiples of 3 or 5 is %d",sum);

  return 0;
}

File: ruby/problem-1.rb
#Project Euler problem 1
#Find the sum of all the multiples of 3 or 5 below 1000.

sum = 0

(1..999).each do |i|
  if(i % 3 == 0 || i % 5 == 0)
    sum += i
  end
end

puts sum #233168
